frontend/src/app/(protected)/experiments/[experimentId]/components/Charts.tsx

```diff
@@ -0,0 +1,240 @@
+import {
+  AreaChart,
+  Area,
+  Tooltip,
+  XAxis,
+  YAxis,
+  ResponsiveContainer,
+} from "recharts";
+import { BetaParams, GaussianParams } from "../types";
```

Comment:
import seems to be wrong

```suggestion
import { BetaParams, GaussianParams } from "../../types";
```

Reply:
Ah yes. I moved it and forgot to correct this. Thank you